common/lib/xmodule/xmodule/js/src/capa/display.coffee

```diff
@@ -451,6 +454,48 @@ class @Problem
       element.CodeMirror.save() if element.CodeMirror.save
     @answers = @inputs.serialize()
 
+  checkAnswersAndCheckButton: (bind=false) =>
```

Comment:
I know that we've been bad about documenting things in the past (especially in capa, :cry: ), but could you document what `bind` does? And about how this method is used in two different contexts (both as setup for listeners and what the listeners call)? It took me a little bit to understand what was going on.